extensions/password-store/src/utils/cmd.util.ts

```diff
@@ -1,5 +1,7 @@
+import { getPreferenceValues } from "@raycast/api";
 import { exec } from "child_process";
 import { promisify } from "node:util";
+import { Preferences } from "../interfaces";
```

Comment:
You don't need to manually set preference types as this is autogenerated in `raycast-env.d.ts`

Can we remove and rely on the auto generated?